AppAuth-JS/app/redirect.html

```diff
@@ -0,0 +1,25 @@
+<!DOCTYPE html>
```

Comment:
add wso2 license here.

Reply:
Google's lib fork, did not change this file, thus keeping the same